hasher-matcher-actioner/terraform/main.tf

```diff
@@ -403,7 +403,8 @@ module "api" {
       api_auth = "hmalib.lambdas.api.api_auth.lambda_handler"
     }
   }
-  datastore = module.datastore.primary_datastore
+  datastore       = module.datastore.primary_datastore
```

Comment:
nit: also add usage widget to dashboard?

Reply:
No, the present dashboard is all about pipeline metrics and latencies. Would like to keep it at that. Once we understand enough about the banks flow, we can see if there is merit in graphing it. Even if there is, should be in a separate dashboard.